### ISSUE TITLE
No copy/paste UI to click in query editor

### ISSUE THREAD

**Issue by gregesposito** [phpmyadmin/phpmyadmin#13849]
### Steps to reproduce
1. Start to make or inline edit an SQL query
2. Right click to copy/paste a value
3. No copy/paste in UI
NOTE: Control-C/Control-V work as expected

### Expected behaviour
Cut/copy/paste etc should be usable via right click
NOTE: It works in the Bookmark the SQL query box shown as expected behavior.
![other-copy-paste](https://user-images.githubusercontent.com/8727407/33390400-cc4d4b2e-d503-11e7-925c-e7ebed0e658b.PNG)

### Actual behaviour
No copy paste options appear
![no-copy-paste](https://user-images.githubusercontent.com/8727407/33390362-b4403c44-d503-11e7-9f58-13344df84c11.PNG)

### Server configuration
Amazon Linux 09.2017 LAMP stack, all packages updated

**phpMyAdmin version:** 4.7.3

### Client configuration
**Browser:** Chrome

**Operating system:** Windows 10

**Comment by nijel** [comment]
Probably somehow related to CodeMirror integration, see also https://github.com/codemirror/CodeMirror/issues/3238

**Comment by ibennetch** [comment]
Even the demo at http://codemirror.net/index.html behaves rather inconsistently with regards to this (I can get the expected menu to appear only after clicking in the textarea then clicking elsewhere on the textarea before right-clicking.

**Comment by nijel** [comment]
For me it works fine in the CodeMirror demo, but not in phpMyAdmin. Tested in Firefox and Chrome, both on Linux.

**Comment by adampurnell** [comment]
My own personal testing on 4.7.6 on both Mac and Windows where applicable.

Chrome 62: Not working on PMA, working on CodeMirror
Firefox Quantum: Not working in PMA, partially working on CodeMirror with a second right click.
Edge: Working perfectly.
Safari: Not working on PMA, working on CodeMirror

**Comment by yashrajbothra** [comment]
![image](https://user-images.githubusercontent.com/47683556/73014473-42564300-3e40-11ea-9f81-b5471055eb3a.png)

please help me reproduce this issue?

**Comment by williamdes** [comment]
#15825 ?

**Comment by gregesposito** [comment]
Hmm, this functionality is working on PMA 4.8.5 on Chrome 79.0.3945.130 (64-bit) on my setup now.

However, it is not working on the PMA demo page running 5.1.0-dev Git revision: d7b3d9f from master branch.

**Comment by uneasyguy** [comment]
https://github.com/uneasyguy/phpmyadmin/pull/1/commits/52962b0f63c49eacca59c3943c1d5aab8fee0198

This is hacky, but solves it for me

**Comment by williamdes** [comment]
Thank you for sharing your fix @gregesposito 

cc @marijnh -> https://github.com/uneasyguy/phpmyadmin/commit/52962b0f63c49eacca59c3943c1d5aab8fee0198

@gregesposito it's probaly best to ping CodeMirror or open an issue and PR: https://github.com/codemirror/CodeMirror

**Comment by uneasyguy** [comment]
@williamdes of course, hope it helps. The real fix would likely entail determining what's unique about certain device/browser setups to cause the function that locates the cursor position to be slightly off, but figured that'd be best left to those who are more familiar with the codemirror code base and/or how it's implemented as it pertains to phpmyadmin, than myself. So hopefully it helps in that respect.
